src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,284 @@
+        printSeparationLine();
+        System.out.println(MARK_AS_DONE_MESSAGE);
+        System.out.println(LONG_SPACE + "[" + thisTask.getStatusIcon() + "] " +
+                thisTask.getDescription());
+        printSeparationLine();
+    }
+
+    static void deleteTask(String userInput) {
+        int selectedTask = Integer.parseInt(userInput.substring(7)) - 1;
+        Task thisTask = tasks.get(selectedTask);
+        System.out.println(REMOVE_TASK_MESSAGE);
+
+        tasks.remove(selectedTask);
+        numberOfTasks--;
+        addTask = false;
+        echo(thisTask, addTask);
+    }
+
+    static void displayList() {
+        printSeparationLine();
+        switch (numberOfTasks) {
+        case 0:
+            System.out.println(EMPTY_LIST_MESSAGE);
+            break;
+        case 1:
+            System.out.println(ONE_TASK_IN_LIST_MESSAGE);
+            break;
+        default:
+            System.out.println(MORE_THAN_ONE_TASK_IN_LIST_MESSAGE);
+        }
+
+        for (int i = 0; i < numberOfTasks; i++) {
+            System.out.println(SHORT_SPACE + (i + 1) + "." + tasks.get(i));
+        }
+        printSeparationLine();
+        System.out.println();
+    }
+
+    static void greet() {
+        System.out.println(GREET_MESSAGE);
+    }
+
+    static void printSeparationLine() {
+        System.out.println(SEPARATION_LINE);
+    }
+
+    static void echo(Task newTask, boolean addTask) {
+        printSeparationLine();
+
+        if (addTask) {
+            System.out.println(ADD_TASK_MESSAGE);
+        } else {
+            System.out.println(REMOVE_TASK_MESSAGE);
+        }
+
+        System.out.println(LONG_SPACE + newTask);
+        if (numberOfTasks == 1) {
+            System.out.println(LONG_SPACE + "Now you have " + numberOfTasks + " task in the list.");
+        } else {
+            System.out.println(LONG_SPACE + "Now you have " + numberOfTasks + " tasks in the list.");
+        }
+
+        printSeparationLine();
+    }
+
+    static void exit() {
+        System.out.println(EXIT_MESSAGE);
+        System.exit(0);
+    }
+
+    private static void readFileContents(String filePath) throws FileNotFoundException {
+        File file = new File(filePath);
+        Scanner sentence = new Scanner(file);
+
+
+        while (sentence.hasNext()) {
+            String[] sentences = sentence.nextLine().split("( \\| )");
+
+            boolean isDone;
+            if (sentences[1] == "1") {
+                isDone = true;
+            } else {
+                isDone = false;
+            }
+            Task task = new Task("");
+            switch (sentences[0]) {
+            case "T":
+                task = new Todo(sentences[2], isDone);
+                break;
+            case "D":
+                task = new Deadline(sentences[2], sentences[3], isDone);
+                break;
+            case "E":
+                task = new Event(sentences[2], sentences[3], isDone);
+                break;
```

Comment:
may want to handle the default case
Also consider storing 'T', 'D', 'E' into their own constants

Reply:
I see! Will implement it in the next commit!